### DISCUSSION TITLE
Support for passing custom icons when rendering GeoJSON

### DISCUSSION THREAD

**Discussion by yass0016** [Canadian-Geospatial-Platform/geoview#1025] [General]
Hi @jolevesq are we able now to pass a custom style (icon) when rendering GeoJSON?

**Comment by jolevesq** [comment]
Hi @yass0016, We do not have a sample for this in our GeoJSON layer but with our vector class (https://canadian-geospatial-platform.github.io/geoview/public/vector.html) we are able to add a marker icon.

Long time no speak, hope everything is going well on your side. I haven't moved your 221-external-package sample yet. I am trying to put more time in coding in the upcoming week. I will try to make this. Thanks
